pkg/chunk/chunk_store.go

```diff
@@ -44,13 +45,18 @@ func init() {
 type StoreConfig struct {
 	CacheConfig
 
+	MaxQueryRangeDays uint
+
 	// For injecting different schemas in tests.
 	schemaFactory func(cfg SchemaConfig) Schema
 }
 
 // RegisterFlags adds the flags required to config this to the given FlagSet
 func (cfg *StoreConfig) RegisterFlags(f *flag.FlagSet) {
 	cfg.CacheConfig.RegisterFlags(f)
+	f.UintVar(&cfg.MaxQueryRangeDays, "max-query-range-days",
```

Comment:
Thanks ... simpler and more conventional
+ I'll do something about the    days    suffix which no longer fits